tools/tertiary-analysis/scanpy/scanpy_macros2.xml

```diff
@@ -1,10 +1,12 @@
 <macros>
-  <token name="@TOOL_VERSION@">1.8.1</token>
+  <token name="@TOOL_VERSION@">1.8.1.1</token>
```

Comment:
I have seen unusual version formatting make the toolshed / galaxy dizzy (for instance not presenting it as the latest version). Maybe we can use a semantic versioning standard for patch (I think it goes with a - after the typical semversion)?

Reply:
Okay, will do

Reply:
Or if you want just bump the +galaxy.

Reply:
Actually it's '+' for a build apparently

Reply:
I feel like the +galaxy should really be for changes in the XML files.

Reply:
Sure, lets go with this.